bindings/go/fdb-go-install.sh

```diff
@@ -272,17 +296,12 @@ else
             # Do not install if only downloading
             :
         elif [[ "${status}" -eq 0 ]] ; then
-            cgo_cflags="-g -O2 -I${linkpath}/bindings/c"
-            cgo_ldflags="-g -O2 -L${FDBLIBDIR}"
+            cgo_cppflags="-I${linkpath}/bindings/c"
+            cgo_cflags="-g -O2"
```

Comment:
Don't the C flags need to have the include path for the C headers? Maybe I'm missing something, though, if this works.

Reply:
The include path is a pre-processor flag; CPPFLAGS is the canonical variable for such flags.

Reply:
Oh, I see. I was auto-completing "CPP" to "C-plus-plus" instead of "C-pre-processor" in my head, but that makes more sense.

Reply:
Ha! All things C++ use the CXX prefix (CXX for the compiler, CXXFLAGS for C++ compiler flags) but still use CPPFLAGS for the pre-processor.